src/liblsquic/lsquic_di_error.c

```diff
@@ -9,24 +9,17 @@
 #include <assert.h>
 #include <stddef.h>
 #include <stdint.h>
+#ifdef WIN32
+#include <vc_compat.h>
+#endif
 
 #include "lsquic_data_in_if.h"
 
 
-static const struct data_in_iface di_if_error;
 
 
-static const struct data_in error_data_in = {
-    .di_if    = &di_if_error,
-    .di_flags = 0,
-};
 
 
-struct data_in *
-data_in_error_new (struct lsquic_conn_public *conn_pub)
-{
-    return (struct data_in *) &error_data_in;
-}
 
```

Comment:
These definitions and functions were moved to after where df_if_error was defined.  I recommend that they remain moved.

Reply:
The problem with moving things around is that it makes source annotation harder.